### DISCUSSION TITLE
Adding Firo (FIRO) as an altcoin on Bisq

### DISCUSSION THREAD

**Discussion by bwusth** [bisq-network/bisq#6138] [Ideas]
I was told that there was a thread here about adding new altcoins, but I have searched and been unable to find it.

I would like to see the privacy coin called **Firo** (FIRO) added as an altcoin possible to trade for BTC on Bisq.

Is there some process for requesting new altcoins?

**Comment by pazza83** [comment]
Hi new asset listing is currently on hold.

**Comment by jeanlescure** [comment]
Any news? Been over a month, would be nice to see Polkadot (DOT) in Bisq as well.
